bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2ScienceAreaGeometry.scala

```diff
@@ -24,10 +24,10 @@ object Igrins2ScienceAreaGeometry extends ScienceAreaGeometry {
 
   val scienceSlitFOV: Shape =
     new Rectangle2D.Double(
-      -ScienceFovWidth.toArcsecs / 2.0,
       -ScienceFovHeight.toArcsecs / 2.0,
-      ScienceFovWidth.toArcsecs,
-      ScienceFovHeight.toArcsecs
+      -ScienceFovWidth.toArcsecs / 2.0,
+      ScienceFovHeight.toArcsecs,
+      ScienceFovWidth.toArcsecs
```

Comment:
Just curious why not swap the values themselves?

Reply:
I did that first but then switcthed back, I guess height is usually larger than width for a slit?